src/biomappings/resources/__init__.py

```diff
@@ -33,12 +44,12 @@ def _load_table(fname) -> List[Dict[str, str]]:
         return [dict(zip(header, row)) for row in reader]
 
 
-def _write_helper(lod: Iterable[Mapping[str, str]], path: str, mode: str) -> None:
+def _write_helper(header: Iterable[str], lod: Iterable[Mapping[str, str]], path: str, mode: str) -> None:
```

Comment:
This should be typed as a `Sequence[str]` now, because using a plain iterable would get exhausted after printing the header